pkg/build/nodeimage/build_impl.go

```diff
@@ -349,6 +329,32 @@ func (c *buildContext) prePullImages(bits kube.Bits, dir, containerID string) ([
 	return importer.ListImported()
 }
 
+func pullImageFns(requiredImages []string, builtImages sets.String, imagesDir string, pulledImages chan string, logger log.Logger) []func() error {
```

Comment:
I'm -1 on this, this loses a ton of context, this method takes way too many parameters.

this whole file does need refactoring, but I'd rather not just do enough to silence some linter.
we're going to continue restructuring build to solve some open issues as-is.

Reply:
yeah... I thought this might be controversial

I'll revert and mark ignore for this method